src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -845,10 +845,242 @@ function bp_core_admin_debug_information( $debug_info = array() ) {
+		),
+		'BP_GROUPS_SLUG' => array(
+			'label' => 'BP_GROUPS_SLUG (deprecated)',
+			'value' => defined( 'BP_GROUPS_SLUG' ) ? BP_GROUPS_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_GROUPS_SLUG' ) ? BP_GROUPS_SLUG : 'undefined',
+		),
+		'BP_MESSAGES_SLUG' => array(
+			'label' => 'BP_MESSAGES_SLUG (deprecated)',
+			'value' => defined( 'BP_MESSAGES_SLUG' ) ? BP_MESSAGES_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_MESSAGES_SLUG' ) ? BP_MESSAGES_SLUG : 'undefined',
+		),
+		'BP_NOTIFICATIONS_SLUG' => array(
+			'label' => 'BP_NOTIFICATIONS_SLUG (deprecated)',
+			'value' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_NOTIFICATIONS_SLUG' ) ? BP_NOTIFICATIONS_SLUG : 'undefined',
+		),
+		'BP_BLOGS_SLUG' => array(
+			'label' => 'BP_BLOGS_SLUG (deprecated)',
+			'value' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_BLOGS_SLUG' ) ? BP_BLOGS_SLUG : 'undefined',
+		),
+		'BP_FRIENDS_SLUG' => array(
+			'label' => 'BP_FRIENDS_SLUG (deprecated)',
+			'value' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_FRIENDS_SLUG' ) ? BP_FRIENDS_SLUG : 'undefined',
+		),
+		'BP_ACTIVITY_SLUG' => array(
+			'label' => 'BP_ACTIVITY_SLUG (deprecated)',
+			'value' => defined( 'BP_ACTIVITY_SLUG' ) ? BP_ACTIVITY_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_ACTIVITY_SLUG' ) ? BP_ACTIVITY_SLUG : 'undefined',
+		),
+		'BP_SETTINGS_SLUG' => array(
+			'label' => 'BP_SETTINGS_SLUG (deprecated)',
+			'value' => defined( 'BP_SETTINGS_SLUG' ) ? BP_SETTINGS_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_SETTINGS_SLUG' ) ? BP_SETTINGS_SLUG : 'undefined',
+		),
+		'BP_XPROFILE_SLUG' => array(
+			'label' => 'BP_XPROFILE_SLUG (deprecated)',
+			'value' => defined( 'BP_XPROFILE_SLUG' ) ? BP_XPROFILE_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_XPROFILE_SLUG' ) ? BP_XPROFILE_SLUG : 'undefined',
+		),
+		'BP_FORUMS_SLUG' => array(
+			'label' => 'BP_FORUMS_SLUG (deprecated)',
+			'value' => defined( 'BP_FORUMS_SLUG' ) ? BP_FORUMS_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_FORUMS_SLUG' ) ? BP_FORUMS_SLUG : 'undefined',
+		),
+		'BP_SEARCH_SLUG' => array(
+			'label' => 'BP_SEARCH_SLUG (deprecated)',
+			'value' => defined( 'BP_SEARCH_SLUG' ) ? BP_SEARCH_SLUG : __( 'Undefined', 'buddypress' ),
+			'debug' => defined( 'BP_SEARCH_SLUG' ) ? BP_SEARCH_SLUG : 'undefined',
+		),
+	);
+
 	$debug_info['buddypress'] = array(
-		'label'  => __( 'BuddyPress', 'buddypress' ),
-		'fields' => array_merge(
+		'label'       => __( 'BuddyPress', 'buddypress' ),
+		'description' => __( 'These settings or constants can alter where and how parts of BuddyPress are loaded or works.', 'buddypress' ),
+		'fields'      => array_merge(
 			array(
+				'bp_settings' => array(
```

Comment:
What happens to this value when you copy the content for debugging purposes? Is the output right?

Reply:
Confirmed the values are being sent while copying the values.

```
bp_settings: thmarker
bp_constants: thmarker
```

Reply:
Good catch, that's pretty annoying. I'll see if i can remove these.

Reply:
This commit https://github.com/buddypress/buddypress/pull/245/commits/e28ebd1a7d7431831e555ee82a7c1a8d5f9e93f0 is marking these fields as private so that they are not sent in copied debug data.

Reply:
Tks!